netkan/netkan/notifications.py

```diff
@@ -19,7 +19,8 @@ def __init__(self, webhook_id, webhook_token):
                                                adapter=discord.RequestsWebhookAdapter())
 
     def emit(self, record):
-        self.webhook.send(self.format(record))
+        fmt = self.format(record)
+        self.webhook.send(f'```{fmt}```' if "\n" in fmt else fmt)
 
```

Comment:
It also accepts markdown styling, so we could do `f'```python\n{fmt}```'`

Reply:
True, but the strings won't be Python code. Even a Python stack trace isn't valid code such that a syntax highlighter would treat it correctly.

Reply:
It does improve readability of the message even if it isn't strictly Python.